webapp/src/services/api/study.ts

```diff
@@ -48,6 +49,37 @@ export const getStudies = async (): Promise<StudyMetadata[]> => {
   });
 };
 
+export const getWorkspaces = async (): Promise<string[]> => {
+  const res = await client.get(`/v1/private/explorer/_list_workspaces`);
```

Comment:
Any specific reason why we have underscores in the api url ? `/v1/private/explorer/_list_workspaces`
We follow [REST conventions](https://restfulapi.net/resource-naming/), this should be something like `/v1/private/explorer/workspaces`, and if separators are needed we usually use a hyphen "-"